Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -893,10 +894,29 @@ - (void)fetchDefaultToken {
   [self defaultTokenWithHandler:handler];
 }
 
-- (void)defaultTokenWithHandler:(FIRInstanceIDTokenHandler)handler {
-  if (self.isFetchingDefaultToken || self.isDefaultTokenFetchScheduled) {
+- (void)defaultTokenWithHandler:(nullable FIRInstanceIDTokenHandler)aHandler {
+  [self defaultTokenWithRetry:NO handler:aHandler];
```

Comment:
I would prefer not to use `defaultTokenWithRetry:handler:`  everywhere because the `retry` argument is  kind of an implementation detail of `defaultTokenWithHandler:` method.

@chliangGoogle Let me know if you still would prefer to get rid of it, I'll create a separate PR for it then.